### PR TITLE
Create trailers file and get trailers loading on pages

### PR DESCRIPTION
**Pull request by Tristan-Holtz**
## What does this PR do?
This creates the trailers file and gets the trailers loading on the MovieDetails pages

## Why are we doing this? Any context or related work?
This completes an extension we added

## Where should a reviewer start?
trailers.js and MovieDetails.js

https://github.com/Tristan-Holtz/Rancid_Tomatillos/projects

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (HartiganHM) [inline .vscode/settings.json]: Make sure you add `.vscode` to you `.gitignore`

File: src/components/MovieDetails/MovieDetails.js [hunk 2]
```diff
@@ -69,6 +16,15 @@ export const MovieDetails = ({ location, user, ratings, setRatings }) => {
         <div className="overview-container">
           <label className="movie-overview">Overview</label>
           <p className="movie-description">{movie.overview}</p>
+          <iframe
```
Comment (ryan-novak) [inline src/components/MovieDetails/MovieDetails.js]: super awesome trailers!!

File: src/components/MovieDetails/MovieDetails.js [hunk 3]
```diff
@@ -69,6 +16,15 @@ export const MovieDetails = ({ location, user, ratings, setRatings }) => {
...
+            title={movie.title + ' trailer'}
```
Comment (HartiganHM) [inline src/components/MovieDetails/MovieDetails.js]: Use a template literal here.

Comment (Tristan-Holtz) [reply src/components/MovieDetails/MovieDetails.js]: That would definitely look cleaner, good call

File: .vscode/settings.json (hunk 1 above)
Comment (Tristan-Holtz) [reply .vscode/settings.json]: Good catch, that snuck in there fixing a merge conflict

**Pull request merged**